_examples/todos-resource/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/pressly/chi"
+	"github.com/pressly/chi/middleware"
+)
+
+func main() {
+	r := chi.NewRouter()
+
+	r.Use(middleware.RequestID)
+	r.Use(middleware.RealIP)
+	r.Use(middleware.Logger)
+	r.Use(middleware.Recoverer)
+
+	r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("."))
+	})
+
+	// r.Mount("/", usersResource{}.Routes()) // TODO: never gets seen...
+	// r.Mount("/", todosResource{}.Routes())
+
+	r.Append(
+		usersResource{}.Routes(),
+		todosResource{}.Routes(),
+	)
```

Comment:
vojtech [4:54 PM]  
btw: I kind of think that we shouldn't add more methods to the router, we should keep it minimalistic and simple

ie. can't we go with `Mount("/", ...chi.Router)` instead of Append()?